consensus/consensus-types/src/sync_info.rs

```diff
@@ -109,17 +160,25 @@ impl SyncInfo {
 
         ensure!(
             self.highest_quorum_cert.certified_block().round()
-                >= self.highest_commit_cert().certified_block().round(),
+                >= self.highest_ordered_cert().certified_block().round(),
             "HQC has lower round than HCC"
         );
+
+        ensure!(
+            self.highest_ordered_cert().certified_block().round()
+                >= self.highest_ledger_info_round(),
+            "HCC has lower round than HCD"
+        );
+
         ensure!(
-            *self.highest_commit_cert().commit_info() != BlockInfo::empty(),
+            *self.highest_ordered_cert().commit_info() != BlockInfo::empty(),
```

Comment:
we should ensure highest_ledger_info.commit_info() is not empty too

Reply:
resolved